### ISSUE TITLE
a11y support is missing for Search field

### ISSUE THREAD

**Issue by anvk** [anvk/a11yAccordion#11]
Once search criteria is changed there should be ARIA live region which announces how many results are displayed out of Total in the accordeon for what Search criteria.
